### PR TITLE
feat(tree): create new tree component & create project tree

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM~!

File: apps/web/src/services/project/tree/use-project-tree-data.ts [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
+            parentGroupId: value.data.parentGroupInfo?.id,
+        }))),
         treeData: [] as TreeNode[],
-        treeUIMap: {} as TreeOpenMap,
+        treeDisplayMap: {} as TreeDisplayMap,
+        selectedTreeNodeId: '',
     });
+    const fetchProjectGroupList = (parentGroupId?: string): ProjectDataType[] => state.projectGroupData.filter((projectGroup) => projectGroup.parentGroupId === parentGroupId);
+    const fetchProjectList = (parentGroupId?: string): ProjectDataType[] => state.projectData.filter((project) => project.parentGroupId === parentGroupId);
+
```
Comment (yuda110) [inline apps/web/src/services/project/tree/use-project-tree-data.ts:56]: praise: nice

**Review by sulmoJ — APPROVED** [review]

File: apps/web/src/services/project/tree/use-project-tree-data.ts [hunk 2]
```diff
@@ -49,149 +71,103 @@ export const useProjectTreeData = (): UseProjectTreeDataReturnType => {
...
+    const init = (): void => {
+        const initialProjectGroup = fetchProjectGroupList();
+        const initialProject = fetchProjectList();
+        state.treeData = convertItemsToTreeData([...initialProjectGroup, ...initialProject], 0);
+    };
+
     onMounted(async () => {
-        await initProjectTreeData();
+        await allReferenceStore.load('project_group');
+        await allReferenceStore.load('project');
```
Comment (yuda110) [inline apps/web/src/services/project/tree/use-project-tree-data.ts:158]: todo: here!

**Review by yuda110 — APPROVED** [review]
LGTM~ Nice work

**Pull request merged**